### PR TITLE
chore(checks): Rename repo to trivy-checks

### PR DESCRIPTION
**Pull request by simar7**
Fixes: https://github.com/aquasecurity/trivy/issues/5609

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -22,7 +22,15 @@ jobs:
           registry: ghcr.io
           username: ${{ env.GH_USER }}
           password: ${{ secrets.GITHUB_TOKEN }}
-      - name: Deploy to GitHub Packages Container registry
+      - name: Deploy policy bundle to ghcr.io (for backwards compatibility)
+        run: |
+          tags=(latest ${{ env.RELEASE_VERSION}} ${{env.MINOR_VERSION }} ${{ env.MAJOR_VERSION }})
+          for tag in ${tags[@]}; do
+              oras push ghcr.io/aquasecurity/trivy-policies:${tag} \
```
Comment (simar7) [inline .github/workflows/release.yaml:29]: Keeping this hardcoded for trivy-policies bundle as we will also rename the repo (to trivy-checks) but want to keep the bundle for backwards compatibility.

**Comment by simar7** [comment]
@nikpivkin could you take a look? The PR itself is minor but please feel to point if if we've missed any other part of the repo (or elsewhere) that needs to be renamed.

**Comment by simar7** [comment]
I've verified the generation of the checks bundle here https://github.com/aquasecurity/trivy-checks/pkgs/container/trivy-checks

File: avd_docs/aws/cloudfront/AVD-AWS-0013/Terraform.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ Use the most modern TLS/SSL policies available
 ```hcl
  resource "aws_cloudfront_distribution" "good_example" {
    viewer_certificate {
-     cloudfront_default_certificate = false
+     cloudfront_default_certificate = aws_acm_certificate.example.arn
```
Comment (nikpivkin) [inline avd_docs/aws/cloudfront/AVD-AWS-0013/Terraform.md:7]: I think you forgot to synchronize with the main branch.

**Comment by nikpivkin** [comment]
@simar7 Trivy has successfully downloaded the new bundle. There is still some use of the term `policies` in code, but mostly in tests. Should this be changed too, or is it not critical?

**Comment by simar7** [comment]
> @simar7 Trivy has successfully downloaded the new bundle. There is still some use of the term `policies` in code, but mostly in tests. Should this be changed too, or is it not critical?

Yeah I thought to just leave it for now.

File: avd_docs/aws/cloudtrail/AVD-AWS-0015/docs.md [hunk 3]
```diff
@@ -1,15 +1,13 @@
 
-Using Customer managed keys provides comprehensive control over cryptographic keys, enabling management of policies, permissions, and rotation, thus enhancing security and compliance measures for sensitive data and systems.
+Cloudtrail logs should be encrypted at rest to secure the sensitive data. Cloudtrail logs record all activity that occurs in the the account through API calls and would be one of the first places to look when reacting to a breach.
```
Comment (nikpivkin) [inline avd_docs/aws/cloudtrail/AVD-AWS-0015/docs.md:2]: Can you synchronise this and other files?

**Review by nikpivkin — APPROVED** [review]
LGTM

**Pull request merged**